api/ssh.go

```diff
@@ -0,0 +1,53 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type Ssh struct {
+	c *Client
+}
+
+func (c *Client) Ssh() *Ssh {
+	return &Ssh{c: c}
+}
+
+func (c *Ssh) KeyCreate(role string, data map[string]interface{}) (*Secret, error) {
+	r := c.c.NewRequest("PUT", fmt.Sprintf("/v1/ssh/creds/"+role))
```

Comment:
I don't think you need the sprintf here if you are going to concatenate with a +